homeassistant/components/bluecurrent/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "domain": "bluecurrent",
```

Comment:
As the company name is "Blue Current", the expected domain would be `blue_current`.